### PR TITLE
Microverse Student Portfolio(Works/about section)

### PR DESCRIPTION
**Pull request by tomdan-ai**
### **Microverse Student Portfolio(Works/about section)**
- [ ] Added a new branch
- [ ] Added more images to the image folder
- [ ] Added a new section in HTML
- [ ] Added more styles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -83,6 +83,204 @@ <h1>
...
+              <h2>Tonic</h2>
+              <div class="canopy">
+                <span>CANOPY</span>
+                <ul>
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+              <p class="daily">
+                A daily selection of privately personalized reads; no accounts
```
Comment (ghost) [inline index.html]: - [ ] In your project cards, kindly position your elements correctly. Adjust the padding and spacing to match the original design

image

### Checkout the original design

image

File: index.html [hunk 2]
```diff
@@ -83,6 +83,204 @@ <h1>
         </ul>
       </div>
       </section>
+
+      <!-- Tonic -->
+
+      <section class="grid">
+        <div class="flex">
```
Comment (ghost) [inline index.html]: - [ ] Kindly ensure there is a spacing between the `headline section` and the `projects section` to match the original design

image

### Checkout the original design

image

File: index.html [hunk 3]
```diff
@@ -83,6 +83,204 @@ <h1>
...
+                  <li class="lang-item">css</li>
+                  <li class="lang-item1">javascript</li>
+                </ul>
+                <button class="project-button">See Project</button>
+              </div>
+            </div>
+        </div>
+
+
+        <div class="main-container">
```
Comment (ghost) [inline index.html]: - [ ] Kindly wrap your `about me content` in a `<section>` tag for semantic reasons.

File: index.html [hunk 4]
```diff
@@ -83,6 +83,204 @@ <h1>
...
+            </div>
+            <div class="connect">
+              <a href="#">LET’S CONNECT</a>
+              <div class="intro-card2">
+                <ul class="icon">
+                  <li>
+                    <a href="#"> <img src="img/Vector.svg" /> </a>
+                  </li>
+                  <li>
+                    <a href="#">
```
Comment (ghost) [inline index.html]: - [ ] Kindly ensure there is a spacing between the `work section` and the `about me section` like in the Figma design
- [ ] Work on the positioning and spacing of elements in your `about me section` to match the original design.

image

### Checkout the original design

image

File: index.html [hunk 5]
```diff
@@ -83,6 +83,204 @@ <h1>
...
+                    <h4>Languages</h4>
+                    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd" d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z" fill="#5E6C84"/>
+                    </svg>
+                </li>
+
+            </ul>
+            <ul class="languages">
+                <li class="languages_details">
+                    <img src="img/IMG 5.png" alt="javascript logo">
```
Comment (ghost) [inline index.html]: - [ ] In your languages section, kindly ensure that the language logos are perfect circles to match the original design
- [ ] You can also reduce the padding in the `skills` and `frameworks` sections to match the original design

image

### Checkout the original design
image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @tomdan-ai , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -24,65 +24,276 @@
...
+          </div>
+          <button type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <!--ABOUT-->
+  <section class="about">
+      <div class="about-me flex column">
+        <div class="is-me flex column">
```
Comment (MeqdamAlqudah) [inline index.html:212]: - [ ] Please as the last reviewer said Kindly ensure there is a spacing between the work section and the about me section like in the Figma design
image

File: index.html [hunk 7]
```diff
@@ -24,65 +24,276 @@
...
+        </div>
       </section>
     </main>
+    <!-- Tonic -->
+
+    <section class="grid">
+      <div class="work-card flex-column">
+        <img src="img/IMG1.png" alt="Tonic" />
+        <div class="work-card1">
+          <h2 class="tonic-class">Tonic</h2>
```
Comment (MeqdamAlqudah) [inline index.html:93]: - [ ] Please as the last reviewer said  Kindly ensure there is a spacing between the headline section and the projects section to match the original design 
image

Comment (MeqdamAlqudah) [inline index.html:93]: - [Optional ] Please make sure that the last card in the work section has the card shape as in the design, and as other cards
image

File: index.html [hunk 8]
```diff
@@ -24,65 +24,276 @@
...
+          <div class="canopy">
```
Comment (MeqdamAlqudah) [inline index.html:94]: - [ Optional] Please add some space to the left of the about me section to match the design
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @tomdan-ai , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
image


 ## Highlights
:+1: no linter errors 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AHP15 — APPROVED** [review]
Hi  @tomdan-ai , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
✅ Requested changes implemented
✅ Good commit messages
✅ Good design
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**